nautobot/ipam/templates/ipam/aggregate_list.html

```diff
@@ -1,7 +1,7 @@
 {% extends 'generic/object_list.html' %}
 {% load humanize %}
 
-{% block sidebar %}
+{% block header_extra %}
     <div class="panel panel-default">
```

Comment:
This takes up a lot of space on the page now. I'm thinking maybe it should be changed to something more like the changelog line, something like:

```jinja2
{% block header_extra %}
   Total IPv4 IPs: <span class="badge">{{ ipv4_total|intcomma }}</span>
   Total IPv6 /64s: <span class="badge">{{ ipv6_total|intcomma }}</span>
{% endblock header_extra %}

though admittedly this may just plain go away in 2.0.
```